lib/components/CTACard/CTACard.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  Grid,
+  Paper,
+  Typography,
+  useTheme,
+} from '@apisuite/fe-base'
+
+import Link from '../../components/Link'
+import { CTACardProps } from './types'
+
+const CTACard: React.FC<CTACardProps> = ({
+  ctaCardButtonLabel,
```

Comment:
Use these button link, label as actions check [billing dialog for an example](https://github.com/APISuite/apisuite-billing-extension-ui/blob/e4005d4ae335b43af7eac016fd2de9837f6ce1a8/lib/components/CustomizableDialog/CustomizableDialog.tsx#L21).